sys/shell/commands/sc_nimble_netif.c

```diff
@@ -117,8 +160,19 @@ static int _conn_dump(nimble_netif_conn_t *conn, int handle, void *arg)
     printf(" ");
     bluetil_addr_ipv6_l2ll_print(conn->addr);
 #endif
-    printf(" (%c,%ums,%ums,%i)", role, itvl, sto, (int)desc.conn_latency);
-    puts("");
+
+#if IS_USED(MODULE_NIMBLE_NETIF_EXT)
+    uint8_t phy_rx, phy_tx;
+    res = ble_gap_read_le_phy(conn->gaphandle, &phy_tx, &phy_rx);
+    if (res != 0) {
+        phy_rx = 0;
```

Comment:
Shouldn't this be `phy_rx = 1` instead? I noticed that when using extended api with no extra phys enabled I got:

```
2021-12-14 17:28:43,425 # ble info
2021-12-14 17:28:43,430 # Own Address: CA:5E:99:AE:11:63 -> [FE80::CA5E:99FF:FEAE:1163]
2021-12-14 17:28:43,432 # Supported PHY modes: 1M
2021-12-14 17:28:43,434 #  Free slots: 1/3
2021-12-14 17:28:43,435 # Advertising: yes
2021-12-14 17:28:43,437 # Connections: 1
2021-12-14 17:28:43,443 # [ 0] EC:25:BC:59:48:68 [FE80::EC25:BCFF:FE59:4868] (S,93ms,1650ms,0,N/A)
2021-12-14 17:28:43,449 #      (role, conn itvl, superv. timeout, slave latency, PHY)
2021-12-14 17:28:43,449 # Slots:
2021-12-14 17:28:43,453 # [ 0] state: 0x0022 - GAP-slave L2CAP-server
2021-12-14 17:28:43,456 # [ 1] state: 0x0100 - advertising
2021-12-14 17:28:43,459 # [ 2] state: 0x8000 - unused
```

When since the only supported phy is 1M and they are connected, shouldn'ty it be displaying 1M?

Reply:
This as well is pending.